src/activity-functions-function-notation.xml

```diff
@@ -105,7 +106,8 @@
 
                     <p>Without knowing anything about function notation, you can still use a language you already know, English, to describe information about the train.</p>
 
-                    <p>For instance:<ul>
+                    <p>For instance:</p>
+                    <ul>
```

Comment:
Another `<ul>` with no `<p>`. In this case I would include the `<ul>` in the "For instance:" `<p>`.